control-plane/pkg/reconciler/broker/controller.go

```diff
@@ -83,7 +83,9 @@ func NewController(ctx context.Context, watcher configmap.Watcher, env *config.E
 		)
 	}
 
-	impl := brokerreconciler.NewImpl(ctx, reconciler, kafka.BrokerClass)
+	impl := brokerreconciler.NewImpl(ctx, reconciler, kafka.BrokerClass, func(impl *controller.Impl) controller.Options {
+		return controller.Options{PromoteFilterFunc: kafka.BrokerClassFilter()}
+	})
```

Comment:
These changes were already merged in https://github.com/knative-sandbox/eventing-kafka-broker/pull/1616

Reply:
wanna rebase? :sweat_smile: